.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -72,7 +72,7 @@ Identify/create the bundles that we will release for Kata and enclave-cc.
     If you absolutely cannot use a Kata release,
     you can consider releasing one of these bundles.
 
-- [ ] 3. :eyes: **Create a peer pods release**
+- [ ] 3. :wrench: **Create a peer pods release**
```

Comment:
So we talked in the past about integrating more of the peer pods release into the operator (e.g. peerpod-ctrl), but we've moved away from that and so the peer pods release effectively has a dependency on the operator - as I pin the released version in our code (e.g. https://github.com/confidential-containers/cloud-api-adaptor/pull/2599), so I wonder if we should update this checklist (and the mermaid diagram) by moving this step below the operator bump merge to better reflect what we are doing?

Reply:
Updated the mermaid diagram too btw ptal. Unfortunately it looks kinda goofy now but oh well.